doozerlib/dblib.py

```diff
@@ -123,6 +123,36 @@ def check_missing_db_env_var(self):
         self.runtime.logger.info("Found all environment variables required for db setup.")
         return True
 
+    def select(self, expr, limit=100):
```

Comment:
The "expr" here will get right column names to be projected, right where condition and correct order by clause as well. But it will have wrong table name. [This](https://github.com/openshift/doozer/blob/3651eba338c1792f666945cfedb824a20e587d03/doozerlib/cli/__main__.py#L231) method will mostly require changes. It is expecting older configuration which might not be present with new implementation. Also it is referring to wrong table to fetch data from. The current table name is (log_build).